mapserver/crdppf.map.mako

```diff
@@ -719,170 +719,170 @@ LAYER
   STATUS ON
   CLASSITEM "codegenre"
   TEMPLATE "ttt"
-    CLASS
+  CLASS
     NAME "Zone de tourisme, sports, détente et loisirs 3 - A constructibilité restreinte"
     EXPRESSION /5219/
     STYLE
         SYMBOL "hatchsymbol"
-        COLOR 100 125 190
-        SIZE 6
-        WIDTH 3
-        ANGLE 0
+        COLOR 100 125 190
+        SIZE 6
+        WIDTH 3
+        ANGLE 0
+        OPACITY 50
+    END #STYLE
+  END #CLASS
+  CLASS
+    NAME "Zone réservée communale"
+    EXPRESSION ([codegenre] = 5912 AND '[teneur]' eq 'Zone réservée communale')
+    STYLE
+        SYMBOL "square"
+        COLOR 64 64 64
+        SIZE 3
+        GAP 20
+    END
+    STYLE
+        OUTLINECOLOR  64 64 64
+        WIDTH 1
+    END #STYLE
+  END #CLASS
+  CLASS
+    NAME "Zone réservée cantonale"
+    EXPRESSION ([codegenre] = 5912 AND '[teneur]' eq 'Zone réservée cantonale')
```

Comment:
Isn't it to mix full expression definitions with classitem definitions (line 764)? Wouldn't it be better to have everything in full expression in this case?

Reply:
classes became more complexe with the new amenagement and dangers naturels structure, thus the full expressions.
I could clean up and update all classes to full expression and latest mapfile definitions. But that would be even more time consuming. So I hesitate...

Any thoughts/recommandations?

Reply:
Nope, it is just strange to see that mix... but there is no good/rapid solution with such a long mapfile...

Reply:
ok, thanks. Then I will gradually adapt all layers for the new version